Code/GraphMol/PeriodicTable.cpp

```diff
@@ -35,9 +35,16 @@ PeriodicTable::PeriodicTable() {
        ++token) {
     if (*token != " ") {
       atomicData adata(*token);
-      byanum.push_back(adata);
       std::string enam = adata.Symbol();
       byname[enam] = adata.AtomicNum();
+      // there are, for backwards compatibility reasons, some duplicate rows for
+      // atomic numbers in the atomic_data data structure. It's ok to have
+      // multiple symbols map to the same atomic number (above), but we need to
+      // be sure that we only store one entry per atomic number.
```

Comment:
I'm in agreement with @ptosco, I would add a small comment like // n.b. AtomicNum 0 is a dummy atom :)

Reply:
that's done